### PR TITLE
LB-303: Add instruction to add last.fm api key to custom_config.py to developement environment setup guide

### PR DESCRIPTION
**Pull request by Uditgulati**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [X] Other

There was no instruction given for adding last.fm api to custom_config.py file in the development environment setup guide.

# Problem

* JIRA ticket (_optional_): [LB-303](https://tickets.metabrainz.org/browse/LB-303)

# Solution

Added the instructions to the guide.

### CODE REVIEW

File: docs/dev/devel-env.rst [hunk 1]
```diff
@@ -111,6 +111,17 @@ Update the strings with your client ID and secret. After doing this, your
...
 
+Also, in order for last.fm import to work, you should also update your last.fm api key in this file. Look for the following section in the file.
+
+.. code-block:: yaml
+
+    # Lastfm API
+    LASTFM_API_URL = "https://ws.audioscrobbler.com/2.0/"
+    LASTFM_API_KEY = "USE_LASTFM_API_KEY"
+
+Update the last.fm api key with your key. After doing this, your
```
Comment (paramsingh) [inline docs/dev/devel-env.rst]: I think we should add a explanation on where to get this key from.

File: docs/dev/devel-env.rst [hunk 2]
```diff
@@ -111,6 +111,17 @@ Update the strings with your client ID and secret. After doing this, your
 ListenBrainz development environment is able to authenticate and log in from
 your MusicBrainz login.
 
+Also, in order for last.fm import to work, you should also update your last.fm api key in this file. Look for the following section in the file.
```
Comment (paramsingh) [inline docs/dev/devel-env.rst]: We try to write `Last.FM` instead of `last.fm` in documentation.

File: docs/dev/devel-env.rst (hunk 1 above)
Comment (Uditgulati) [reply docs/dev/devel-env.rst]: Just adding link to API form would be enough or should i add something else too?

File: docs/dev/devel-env.rst (hunk 2 above)
Comment (Uditgulati) [reply docs/dev/devel-env.rst]: Ok :+1:

File: docs/dev/devel-env.rst (hunk 1 above)
Comment (paramsingh) [reply docs/dev/devel-env.rst]: You can obtain a Last.FM API key from here: <link>

**Comment by paramsingh** [comment]
@mayhem, you should take a look at the verbiage here once, IMO.

**Comment by mayhem** [comment]
It doesn't flow super well, but really, there is nothing here stopping this from being merged.

**Pull request merged**